src/js/veteran-id-card/containers/Main.jsx

```diff
@@ -58,8 +74,8 @@ class Main extends React.Component {
 
     if (this.props.errors) {
       message = this.renderErrors();
-    } else if (this.props.redirect) {
-      window.location.href = this.props.redirect;
+    } else if (this.props.vicUrl) {
+      this.submitVicForm();
```

Comment:
Generally, we try not to trigger actions in the render method and let it only render things. It would be better to put this in one of the lifecycle methods, probably `componentDidUpdate()`.